draft/ZEP0006.md

```diff
@@ -0,0 +1,274 @@
+---
+layout: default
+title: ZEP0006
+description: Zarr Object Models (ZOMs)
+parent: draft ZEPs
+nav_order: 1
```

Comment:
```suggestion
nav_order: 6
```

Reply:
Hi @d-v-b. This would allow the ZEP to be listed in chronological order.

Reply:
thanks @MSanKeys963 , I will make this change